src/System.IO.FileSystem/tests/Performance/Perf.FileStream.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Perf_FileStream : FileSystemTest
+    {
+        private static byte[] CreateBytesToWrite(int size)
+        {
+            PerfUtils utils = new PerfUtils();
+            return Text.Encoding.UTF8.GetBytes(utils.CreateString(size));
+        }
+
+        private string CreateFile(int size)
+        {
+            PerfUtils utils = new PerfUtils();
+            string filePath = GetTestFilePath();
+            byte[] bytes = Text.Encoding.UTF8.GetBytes(utils.CreateString(size));
+            File.WriteAllBytes(filePath, bytes);
+            return filePath;
+        }
+
+        [Benchmark]
+        [InlineData(2000000)]
+        [InlineData(100000000)]
+        public void Write(int size)
+        {
+            string filePath = GetTestFilePath();
+            byte[] bytes = CreateBytesToWrite(size);
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (FileStream writer = File.Create(filePath))
+                using (iteration.StartMeasurement())
+                {
+                        writer.Write(bytes, 0, size);
```

Comment:
I'm not sure we need the 2nd case, but the 1st and 3rd sound good.  You might also consider one for ReadByte, which is essentially size == 1.